changedetectionio/tests/test_import.py

```diff
@@ -201,3 +201,4 @@ def test_import_watchete_xlsx(client, live_server):
             filters = watch.get('include_filters')
             assert filters[0] == '/html[1]/body[1]/div[4]/div[1]/div[1]/div[1]||//*[@id=\'content\']/div[3]/div[1]/div[1]||//*[@id=\'content\']/div[1]'
             assert watch.get('time_between_check') == {'weeks': 0, 'days': 1, 'hours': 6, 'minutes': 24, 'seconds': 0}
+            assert watch.get('fetch_backend') == 'html_requests'
```

Comment:
I see this test is failing. In test `fetch_backend` is setting to `system` and not `html_requests` as I expect. However, when I tested manually by uploading same test file spreadsheet.xlsx `fetch_backend` is setting correctly to `html_requests`. @dgtlmoon Do you have any insight on this?

Reply:
When only test `test_import_watchete_xlsx` runs, it passes.

When all tests run, test `test_import_watchete_xlsx` is using values from previous test `test_import_custom_xlsx`.

I am not sure correct way to isolate both of these tests. This is first time I ran tests in Python and also new to changedetection.io.

1. We may use separate files `import/spreadsheet.xlsx` with different data so that imports are isolated.
2. Before tests run, we may delete all existing watches.
3. Any other thought?

Reply:
> 2. Before tests run, we may delete all existing watches.

Went ahead with this approach and shamelessly copied following code from other tests to cleanup :)

```
    # Cleanup everything
    res = client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
    assert b'Deleted' in res.data
```
All 5 tests of `tests/test_import.py` are now passing.